src/parser/core/modules/Overheal.tsx

```diff
@@ -225,6 +223,7 @@ export class CoreOverheal extends Module {
 		const guid = event.ability.guid
 		for (const trackedHeal of this.trackedOverheals) {
 			if (trackedHeal.idIsTracked(guid)) {
+				this.debug(`Heal from ${event.ability.name} at ${event.timestamp} matched into category ${trackedHeal.name.props.defaults}`)
```

Comment:
Can you go ahead and duplicate this debug hook for "matched into Direct" as well down below, to make it easier for future maintainers to see what's going on?

Also, one thing that showing the matched into Direct reveals is that the "Regeneration" events for passive regeneration are also matching into that bucket.  I think that should probably be excluded, as it will make overheal from direct heal actions appear a bit lower than it really was, since the player doesn't really control their own self-Regeneration.

Reply:
the duplicate is honestly a good idea. I pushed more changes in and added Ids just for additional tracking purposes.